### ISSUE TITLE
add mob member by url, to reduce friction point, and make it easier to mob with others

### ISSUE THREAD

**Issue by alterisian** [mobtimeapp/mobtime#285]
So That: I can easily get involved with the current mob
As A: Mob Organiser
I Want To: Send a url to a potential collaborator, that allows that to fill in a text field with their name, adding them to the mob.
-- 
Notes: Is this what is wanted? Discussion: https://twitter.com/alterisian/status/1623294428109582336

**Comment by mrozbarry** [comment]
So the idea here is a url could be given that would have a user enter their name before the timer loads?

**Comment by alterisian** [comment]
Yeah. That's it. Basically to make it easier for people to be invited, and join the mob. 
One friction point removed a bit. 

I initially thought that people could send this by whatsapp, twitter, email etc to potential MobInvitees. But it could probably also go into a calendar for yourself. Then the reminder would have the how to get involved in the mob, they put their name into buy into it, and the video/jitsi/pop/etc url on how to start, then look for the goals.
